components/script/dom/htmlmetaelement.rs

```diff
@@ -113,4 +113,11 @@ impl VirtualMethods for HTMLMetaElement {
             self.process_attributes();
         }
     }
+
+    fn parse_plain_attribute(&self, name: &Atom, value: DOMString) -> AttrValue {
```

Comment:
Ah, sorry, @jdm and I were so excited when we finally got Servo to build on my Mac that I didn't push the import I made to get it working--I'll push that now.